src/verifast1.ml

```diff
@@ -2713,6 +2713,7 @@ module VerifyProgram1(VerifyProgramArgs: VERIFY_PROGRAM_ARGS) = struct
   let rec type_satisfies_contains_any_constraint allowContainsAnyPositive tp =
     match unfold_inferred_type tp with
       Bool | AbstractType _ | Int (_, _) | Float | Double | LongDouble | RealType | FuncType _ | PtrType _ | ObjType _ | ArrayType _ | BoxIdType | HandleIdType -> true
+(*    | StructType (_, ts) -> List.for_all (fun t -> type_satisfies_contains_any_constraint allowContainsAnyPositive t) ts *)

```

Comment:
A struct type `struct S<T1,...,Tk> { f1: t1, ..., fN: tN }` should be treated like an `inductive S<T1,...,Tk> = S(t1, ..., tN);`